test/base/resources/eventing.go

```diff
@@ -118,6 +118,32 @@ func WithTriggerFilter(eventSource, eventType string) TriggerOption {
 	}
 }
 
+// WithAttributesTriggerFilter returns an option that adds a TriggerFilter with Attributes for the given Trigger.
+func WithAttributesTriggerFilter(eventSource, eventType string) TriggerOption {
```

Comment:
I think you can call you can just add an `extensions map[string]string` parameter, and get rid of the method below.